lib/processor.js

```diff
@@ -170,23 +166,26 @@ function parse(text) {
 
         // Line content
         } else {
-            next = match(/[^\r\n]*/);
+            next = match(/^[^\r\n]*/);
```

Comment:
Oh, ok.  It was `text` that was the whole text up to the end of file.  Not sure why I didn't think to use `next`.